static/scss/pages/dashboard.scss

```diff
@@ -149,6 +158,10 @@
     position: relative;
     z-index: 1;
     margin-left: $spacing-sm;
+
+    img {
```

Comment:
nit: does this `img` class need to be repeated here if the `mpp-dashbaord-header-action` parent has the same one?

Reply:
Good call – I actually need to add an additional line (`margin-right: 0`) to overwrite the other rule. This image comes _after_ the copy, where the other image comes before.